contracts/wawaka/exchange/contracts/common/Asset.h

```diff
@@ -0,0 +1,68 @@
+/* Copyright 2019 Intel Corporation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include "Value.h"
+
+#include "Common.h"
+#include "StateReference.h"
+
+#define ASSET_SCHEMA "{"                   \
+    SCHEMA_KW(asset_type_identifier,"") ","     \
+    SCHEMA_KW(count,0) ","                      \
+    SCHEMA_KW(owner_identity,"") ","            \
+    SCHEMA_KW(escrow_agent_identity,"") ","     \
+    SCHEMA_KW(escrow_identifier,"")             \
```

Comment:
no. it is any entity to which i want to grant temporary ownership of the assets. for example, it will often be the fair exchange or auction contracts.

Reply:
Thank you.